index.src.html

```diff
@@ -1692,6 +1711,11 @@
 
 dictionary SvcOutputMetadata {
   unsigned long temporalLayerId;
+  unsigned long spatiallayerId;
+  unsigned short frameNumber;
```

Comment:
Why not frameId like in RTCEncodedVideoFrameMetadata ?

Reply:
As noted in the[ Dependency Descriptor specification section A.8.3](https://aomediacodec.github.io/av1-rtp-spec/#2-conventions-definitions-and-acronyms), frame_number is not the same as FrameId.